docs/v1/Makefile

```diff
@@ -5,22 +5,27 @@ ROUGE_CSS=style
 
 all: ${SPEC}.pdf ${SPEC}.html
 
+folder:
+	mkdir -p build
+
 build:
 
-${SPEC}.pdf: ${SPEC}.adoc images
+${SPEC}.pdf: ${SPEC}.adoc images folder
```

Comment:
what is `folder`, the name of some directory?

Reply:
It is the name of a target elsewhere in this same Makefile, like `images` is.

Reply:
Oh DUH! `folder` is right above. `ERROR 1, SYSTEM RESOURCES LOW: INSUFFICIENT COFFEE ON SATURDAY MORNING!`